Code/Editor/EditorViewportSettings.cpp

```diff
@@ -186,6 +187,16 @@ namespace SandboxEditor
         AzToolsFramework::SetRegistry(ManipulatorLineBoundWidthSetting, lineBoundWidth);
     }
 
+    bool ViewportMouseWrapSetting()
+    {
+        return aznumeric_cast<bool>(GetRegistry(ViewportMouseWrapSetting, false));
```

Comment:
nit: You can remove the `aznumeric_cast` here, it can just be `return GetRegistry(ViewportMouseWrapSetting, false);` (the `aznumeric_cast` is for the `double`/`float` conversions as the Settings Registry uses `double`s